omod/src/main/webapp/pages/labtrackingAddOrder.gsp

```diff
@@ -255,8 +255,8 @@ ${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient ]) }
       .value('patientUuid', '${ patient.uuid }')
       .value('visitUuid', '${ visit.uuid }')
       .value('orderUuid', '${ orderUuid }')
-      .value('visitStartDateTime', '${ visit ? visit.startDatetime : '' }')
```

Comment:
@mogoodrich , here is where the problem with the visit startDatetime missing timezone is. This gsp line visit.startDatetime does not include the timezone. Maybe the java object had the timezone component but that was not marshalled in this gsp page, therefore the need to specifically included as a separate property.